front/playwright.config.ts

```diff
@@ -62,7 +62,7 @@ const config: PlaywrightTestConfig = {
 
   /* select tags to run specific tests */
   // TODO: remove grep when every tests are refactored
-  grep: [/home/, /project/, /study/, /scenario/],
+  grep: [/editor/, /home/, /project/, /scenario/, /study/],
```

Comment:
please respect order : home / project / study / scenario / editor  thx

Reply:
ok updated the order and split in two commits

Reply:
Any reason for this order? I find the alphabetical order more logical to respect. If there is a specific need for another order, maybe a comment would help?

Reply:
in a future pr the grep will be associated with the keyword `enable` only

Reply:
My bad, I'v not seen that it was ordered in alphabetical order (sorry @tykayn!). No passion for me, so do whatyou think that'll better (original order was logical order of views hierarchy, not so obvious!)